tests/structures/test_exception.py

```diff
@@ -85,11 +85,11 @@ def test_raising_exceptions_multiple_args(self):
                 try:
                     raise exc("one")
                 except Exception as e:
-                    print(e)
+                    print(e, e.args)
```

Comment:
Maybe we could have `str(e)` and `repr(e)` also in these tests?

Reply:
Done